Application/EdFi.Ods.AdminApi/E2E Tests/Admin API.postman_environment.json

```diff
@@ -14,18 +14,30 @@
 			"type": "default",
 			"enabled": true
 		},
-    {
-        "key": "limit",
-        "value": "1000",
-        "type": "default",
-        "enabled": true
-    },
-    {
-        "key": "connectionString",
-        "value": "Data Source=.;Initial Catalog=EdFi_Admin;Integrated Security=True;TrustServerCertificate=True",
-        "type": "default",
-        "enabled": true
-    }
+		{
+			"key": "limit",
+			"value": "1000",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "connectionString",
+			"value": "Data Source=.;Initial Catalog=EdFi_Admin;Integrated Security=True;TrustServerCertificate=True",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "Tenant1",
+			"value": "Tenant1",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "Tenant2",
+			"value": "Tenant2",
+			"type": "default",
+			"enabled": true
+		}
```

Comment:
@simpat-adam 
Do we need tenant details here ?

Reply:
Removed.